WPMedia/fusion-news-theme-blocks
# Added option object to respect session across tabs

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

Added option object to identity.login so the SDK will save the user's tokens to the browser SessionStorage, and the user's session will be respected across tabs

## Jira Ticket

- [THEMES-1415](https://arcpublishing.atlassian.net/browse/THEMES-1415)

## Acceptance Criteria

_copy from ticket_

## Test Steps

1. Checkout this branch git checkout THEMES-1415
2. Make sure your feature pack uses the corecomponents content base, and the blocks.json includes:
```
				"api": {
					"identity": {
						"origin": "https://corecomponents-the-gazette-prod.api.cdn.arcpublishing.com"
					}
				}
```
3. Run fusion repo with linked blocks npx fusion start -f -l @wpmedia/identity-block
4. The account management page requires a logged in user to view.
5. Create a page in PageBuilder, one with the 'Identity Log In - Arc Block' 
6. Navigate to the login page you created and use the test login for The Gazette listed here: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/2794422817/QA.
7. Verify that the user was succesffully logged in (the nav bar should show a single "Account" button
8. After logging in, visit other local pages and make sure that the user is still logged in

## Effect Of Changes

### Before

If a new tab was opened, the user would no longer be logged in

### After

If a new tab was opened, the user will remain logged in

## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Manifest pr for new block:
- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Dependency on another PR that needs to be merged first

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1415]: https://arcpublishing.atlassian.net/browse/THEMES-1415?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
